clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -3396,6 +3396,13 @@ LangAS CodeGenModule::GetGlobalVarAddressSpace(const VarDecl *D) {
     return AddrSpace;
   }
 
+  if (LangOpts.SYCLIsDevice) {
+    if (D && D->getType().isConstQualified())
+      return LangAS::sycl_constant;
```

Comment:
should these be LangAS::opencl_constant and LangAS::opencl_global similar to what they are in SemaSYCL.cpp (https://github.com/intel/llvm/blob/sycl/clang/lib/Sema/SemaSYCL.cpp#L741)? I think the intent was to eventually get rid of the sycl specific address spaces and reuse the OpenCL ones or am I perhaps missing something?

Reply:
Makes sense.

Reply:
LGTM!